.evergreen/deploy.yml

```diff
@@ -0,0 +1,96 @@
+functions:
+  setup-credentials:
+    command: shell.exec
```

Comment:
Should we consider rewriting this with `subprocess.exec`?

Reply:
Imo this isn't super straightforward so would prefer to leave it for the ticket that captures that work!